crates/nu-command/tests/commands/drop.rs

```diff
@@ -27,11 +25,11 @@ fn more_columns_than_table_has() {
               [3,  white]
               [8, yellow]
               [4,  white]
-            ] | drop column 3 | columns | empty?
+            ] | drop column 3 | columns | length
```

Comment:
I'd prefer `empty?` work here - there a known problems with `empty?` and we need to fix it.

Reply:
Fine, I will revert it